sdk/recoveryservices-backup/Azure.ResourceManager.RecoveryServicesBackup/api/Azure.ResourceManager.RecoveryServicesBackup.netstandard2.0.cs

```diff
@@ -2588,6 +2594,18 @@ public partial class TargetAfsRestoreInfo
         public string Name { get { throw null; } set { } }
         public Azure.Core.ResourceIdentifier TargetResourceId { get { throw null; } set { } }
     }
+    public enum TargetDiskNetworkAccessOption
+    {
+        SameAsOnSourceDisks = 0,
+        EnablePrivateAccessForAllDisks = 1,
+        EnablePublicAccessForAllDisks = 2,
+    }
+    public partial class TargetDiskNetworkAccessSettings
+    {
+        public TargetDiskNetworkAccessSettings() { }
+        public string TargetDiskAccessId { get { throw null; } set { } }
```

Comment:
Is the value of this property always an arm resource id?
If so, please change the format of it to `arm-id`

Reply:
updated as per suggestion